### DISCUSSION TITLE
QTable columns default align

### DISCUSSION THREAD

**Discussion by robsontenorio** [quasarframework/quasar#11886] [Ideas / Proposals]
Why current `QTable` columns default align is `right`?

Most of systems everything on tables are placed on `left`. For sure, number/money columns fits better on right. But left align is the 90% of default placement.

**Comment by MilosPaunovic** [comment]
As to what prompted this decision, I can't answer. TBH, it would work better for me, as well, if the default alignment was to the left. 

This would be easy to change, in this line: https://github.com/quasarframework/quasar/blob/4b36ed6a75e635b3977c3313b9e77f21d91a2618/ui/src/components/table/table-column-selection.js#L37

But the question is do the authors want that.
If so, I could submit a PR to make thing easier, if not, we'll get by with `align` prop without problems.

**Reply by pdanpdan** [reply]
It cannot be changed because it would mess all people that have not set it

**Reply by MilosPaunovic** [reply]
Understood 🙂 

Just curious, was there any particular reason behind the decision to choose `right` as the default or was it more of a judgement call for more colums to contain numbers than not, cause of the [Material](https://material.io/components/data-tables#specs) numeric alignment specs?

**Comment by robsontenorio** [comment]
Well , maybe v3 where breaking changes are expected. 

Thanks for answers.
